config/webpack/environment.js

```diff
@@ -1,3 +1,15 @@
 const { environment } = require('@rails/webpacker')
+const webpack = require('webpack');
+
+environment.plugins.append('Provide', new webpack.ProvidePlugin({
+   $: 'jquery',
+   jQuery: 'jquery'
+}));
+
+const config = environment.toWebpackConfig();
+
+config.resolve.alias = {
```

Comment:
Would you mind added a comment in the file explaining why `config.resolve.alias = {     jquery: 'jquery/src/jquery'  } ` is needed?

Reply:
I was thinking we should perhaps treat JQuery like any other library and just import it in the application file. It seems cleaner and a bit more transparent.